external-docs/docs/guides/errors/build-command-errors/duplicate-measures.md

```diff
@@ -0,0 +1,11 @@
+# Error - Duplicate Measures Error
```

Comment:
Can you follow the capitalisation pattern we have for titles in the external user documentation? The first word should be capitalised, and the following words should all be lower case unless using acronyms. I think this affects all of your new docs pages.

Reply:
Changed titles to match capitalisation pattern in each new documentation page.